packages/webdriverjs/package.json

```diff
@@ -1,37 +1,19 @@
 {
   "name": "@axe-core/webdriverjs",
   "version": "4.1.1",
-  "description": "Provides a method to inject and analyze web pages using aXe",
+  "description": "Provides a method to inject and analyze web pages using axe",
+  "author": "Michael Siek<me@michaelsiek.com>",
```

Comment:
I think it's fine to add yourself to `contributors`, but claiming ownership of this package probably won't fly...